GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -977,6 +977,9 @@ void ProcessHeads(IReadOnlyList<IGitRef> remoteHeads)
                 var localHeads = GetLocalBranches().ToList();
                 var remoteBranches = remoteHeads.ToHashSet(h => h.LocalName);
 
+                // Fix #7077: Optimize data loading notifications
```

Comment:
This is redundant, this can be tracked via git history. Please remove.

Applies universally.

Reply:
Done